internal/operators/lvm/lvm_operator_test.go

```diff
@@ -0,0 +1,162 @@
+		// 		Disks: []*models.Disk{
+		// 			{
+		// 				SizeBytes: 20 * conversions.GB,
+		// 				DriveType: models.DriveTypeHDD,
+		// 				ID:        diskID1,
+		// 			},
+		// 			{
+		// 				SizeBytes: 40 * conversions.GB,
+		// 				DriveType: models.DriveTypeSSD,
+		// 				ID:        diskID2,
+		// 			},
+		// 		},
+		// 	}),
+		// }
+		// hostWithInsufficientRAM = &models.Host{
+		// 	InstallationDiskID: diskID1,
+		// 	Inventory: Inventory(&InventoryResources{
+		// 		Cpus: 12,
+		// 		Ram:  1199 * conversions.MiB,
+		// 		Disks: []*models.Disk{
+		// 			{
+		// 				SizeBytes: 20 * conversions.GB,
+		// 				DriveType: models.DriveTypeHDD,
+		// 				ID:        diskID1,
+		// 			},
+		// 			{
+		// 				SizeBytes: 40 * conversions.GB,
+		// 				DriveType: models.DriveTypeSSD,
+		// 				ID:        diskID2,
+		// 			},
+		// 		},
+		// 	}),
+		// }
+		hostWithSufficientResources = &models.Host{
+			InstallationDiskID: diskID1,
+			Inventory: Inventory(&InventoryResources{
+				Cpus: 12,
+				Ram:  32 * conversions.GiB,
+				Disks: []*models.Disk{
+					{
+						SizeBytes: 20 * conversions.GB,
+						DriveType: models.DriveTypeHDD,
+						ID:        diskID1,
+					},
+					{
+						SizeBytes: 40 * conversions.GB,
+						DriveType: models.DriveTypeSSD,
+						ID:        diskID2,
+					},
+				},
+			}),
+		}
+	)
+
+	Context("GetHostRequirements", func() {
+		table.DescribeTable("get host requirements when ", func(cluster *common.Cluster, host *models.Host, expectedResult *models.ClusterHostRequirementsDetails) {
+			res, _ := operator.GetHostRequirements(ctx, cluster, host)
+			Expect(res).Should(Equal(expectedResult))
+		},
+			table.Entry("host",
+				&common.Cluster{Cluster: models.Cluster{Hosts: []*models.Host{hostWithSufficientResources}}},
+				hostWithSufficientResources,
+				&models.ClusterHostRequirementsDetails{CPUCores: operator.config.LvmCPUPerHost, RAMMib: operator.config.LvmMemoryPerHostMiB},
+			),
+		)
+	})
+
+	Context("ValidateHost", func() {
+		table.DescribeTable("validate host when ", func(cluster *common.Cluster, host *models.Host, expectedResult api.ValidationResult) {
+			res, _ := operator.ValidateHost(ctx, cluster, host)
+			Expect(res).Should(Equal(expectedResult))
+		},
+			table.Entry("host with no inventory",
+				&common.Cluster{Cluster: models.Cluster{Hosts: []*models.Host{hostWithNoInventory}}},
+				hostWithNoInventory,
+				api.ValidationResult{Status: api.Pending, ValidationId: operator.GetHostValidationID(), Reasons: []string{"Missing Inventory in the host"}},
+			),
+			table.Entry("host with insufficient disks",
+				&common.Cluster{Cluster: models.Cluster{Hosts: []*models.Host{hostWithInsufficientDisks}}},
+				hostWithInsufficientDisks,
+				api.ValidationResult{Status: api.Failure, ValidationId: operator.GetHostValidationID(), Reasons: []string{"Insufficient disks, ODF LVM requires at least one non-installation disk on the host"}},
+			),
+			// table.Entry("host with insufficient CPU",
```

Comment:
please remove those lines if it's not needed

Reply:
resolved